test/e2e/self_hosted_test.go

```diff
@@ -31,19 +32,39 @@ var _ = Describe("When testing Cluster API working on self-hosted clusters", fun
 			BootstrapClusterProxy: bootstrapClusterProxy,
 			ArtifactFolder:        artifactFolder,
 			SkipCleanup:           skipCleanup,
+			EnableUpgrade:         false,
```

Comment:
What about running the upgrade here as well? Seems relatively efficient to just test a bit more in this case when we're already that far in the e2e test

Reply:
given that this now also works because of using the variable for images preload 👍